src/device/pnet_api.c

```diff
@@ -41,6 +41,13 @@ pnet_t* pnet_init(
       return NULL;
    }
 
+   if (strlen(netif) >= PNET_MAX_INTERFACE_NAME_LENGTH)
```

Comment:
Is this really correct wrt to overflow and termination? What if netif contains exactly PNET_MAX_INTERFACE_NAME_LENGTH characters?

Reply:
Done